### ISSUE TITLE
ListView with GridView fails

### ISSUE THREAD

**Issue by rerdavies** [Kinnara/ModernWpf#114]
ListViews with GridViews fail to display properly by default.

The sample app reveals the workaround:

```
  <ListView x:Name="ErrorList" 
                  Style="{StaticResource {x:Static GridView.GridViewStyleKey}}"
..
  >
       <GridView>
    ....
```

But it would be really nice if you could DOCUMENT this somewhere other than in the sample app source code.

**Comment by Kinnara** [comment]
Thanks for bringing this up. In the next version it'll work by default.

**Comment by Kinnara** [comment]
Fixed by 55f8b5104e5d45811e7f0f84a1694f8ac23fde3e.
